src/main/resources/db/migration/V0_0_1_20240329124018__insert-oidc-registered-client.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO oauth2_registered_client
+(id, client_id, client_id_issued_at, client_secret, client_secret_expires_at,
+client_name, client_authentication_methods, authorization_grant_types,
+redirect_uris, post_logout_redirect_uris,
+scopes,
+client_settings,
+token_settings)
+VALUES('1d0ae4d0-0ec5-4f64-92aa-db46d718ebae', 'oidc-client', '2024-03-28 15:20:33', '{noop}secret', NULL,
+'oidc-client', 'client_secret_post,client_secret_basic', 'refresh_token,client_credentials,authorization_code',
```

Comment:
apply Bcrypt encoder for client secret instead of NoOpPasswordEncoder

Reply:
changed to bcrypt string

Reply:
Have you tested after changing this string?
Do the token issuing still work well?

Reply:
The default password encoder of Oauth2 is Bcrypt Encoder, so we don't need to declare a bean for passwordEncoder.